api/base/serializers.py

```diff
@@ -222,13 +222,25 @@ def __init__(self, related_view=None, related_view_kwargs=None, self_view=None,
         self.view_kwargs = {'related': related_kwargs, 'self': self_kwargs}
         self.related_meta = related_meta
         self.self_meta = self_meta
+        self.always_embed = always_embed
         assert (related_view is not None or self_view is not None), 'Self or related view must be specified.'
         if related_view:
             assert related_kwargs is not None, 'Must provide related view kwargs.'
+            assert isinstance(related_kwargs, dict), "Related view kwargs must have format {'lookup_url_kwarg: lookup_field}."
         if self_view:
             assert self_kwargs is not None, 'Must provide self view kwargs.'
+            assert isinstance(self_kwargs, dict), "Self view kwargs must have format {'lookup_url_kwarg: lookup_field}."
+
+        view_name = related_view
+        if view_name:
+            lookup_kwarg = related_kwargs.keys()[0]
```

Comment:
https://github.com/pattisdr/osf.io/blob/feature/api/includes-2/api/base/views.py#L29-L30

I forgot about this: I was handling multiple kwargs on this line: